web/src/App.vue

```diff
@@ -1,21 +1,49 @@
 <template lang="pug">
 v-app
-  v-toolbar.main-toolbar.darken-4(dark, dense, color="primary")
-    v-toolbar-items
-      v-btn.px-0.square-button.darken-4(flat, color="white", to="/select")
-        v-icon {{ $vuetify.icons.close }}
-    v-spacer
-    v-toolbar-title.title
-      span.font-weight-bold.secondary--text.text--lighten-3 Biomarker Analysis »
-      span.font-weight-bold.white--text  {{ $router.currentRoute.name }}
+  v-toolbar.darken-3(dense, dark, color="primary")
+    v-toolbar-side-icon.logo(@click="$router.push('/')") V
+    v-breadcrumbs(:items="breadcrumbs", divider="»")
+      template(#item="props")
+        v-breadcrumbs-item(:to="props.item.to", exact) {{props.item.text}}
     v-spacer
+    //- v-btn(icon)
+    //-   v-icon {{ $vuetify.icons.settings }}
   v-container.pa-0.d-flex(fluid)
     router-view.grow
 </template>
 
 <script>
+
+function injectParams(path, params) {
+  let parsed = path;
+  Object.entries(params).forEach(([k, v]) => {
+    parsed = parsed.replace(`:${k}`, v);
+  });
+  return parsed;
+}
+
 export default {
   name: 'App',
+  computed: {
+    breadcrumbs() {
+      const toBreadcrumb = (route) => {
```

Comment:
prefer not to nest named functions in functions: either move to `methods` or inline as anonymous on line 43.